### DISCUSSION TITLE
An error occurred while running the personal case

### DISCUSSION THREAD

**Discussion by wuesh** [NCAR/wrf_hydro_nwm_public#766] [Q&A] [answered]
Hello everyone,
I tried to prepare and process the data myself and modify the "hydro-namelist" and "namelist.hrldas" files, but a fatal error occurred after running for a short time. 
“The job is stopped due to the fatal error. FATAL ERROR:  - Problem nf90_create in nc of hydrotab_f file ", I have no idea what is causing the problem.
Any help would be greatly appreciated！
[out.log](https://github.com/user-attachments/files/16537196/out.log)
[Uploading namelist&wrfinput_d01.zip…]()

**Comment by scrasmussen** [comment]
Hi @wuesh, 

Could you modify the following in `Routing/module_HYDRO_io.F` line ~11046 so it is
```
     if (iret /= 0) then
          print *," nf90 error:", trim(nf90_strerror(iret))
          call hydro_stop("FATAL ERROR:   - Problem nf90_create  in nc of hydrotab_f file")
     endif
```

The `nf90_strerror` will report the exact error and give us a better idea of what is going on. - Thanks!

**Reply by wuesh** [reply]
Thank you for your help. I'm a novice, so I made a simple mistake.

**Comment by aubreyd** [comment, marked as answer]
Yes - this is a bit of a hidden feature. If you specify a HYDROTBL_F filename that exists, the code will read that file. If you specify a filename that does NOT exist, the code will read the HYDRO.TBL parameter table and create the 2d netcdf parameter file at the path/filename you specified. So if you are specifying a filename that does not exist, make sure the path does and you have write permission there.

**Reply by wuesh** [reply]
Thank you for your help. The reason for the error is that I did not modify the path of the hydro2dtbl.nc file. I thought that this file would be generated automatically in the current directory if it did not exist, but the fact is that it will be generated in the specified path.
